data/json/vehicle_parts.json

```diff
@@ -3974,6 +3997,2249 @@
         "location" : "on_roof",
         "flags" : ["CIRCLE_LIGHT", "ODDTURN", "FOLDABLE"],
         "breaks_into" : [
+          {"item": "steel_lump", "min": 4, "max": 6},
```

Comment:
Yep, steel lumps are way out of scale for a police light.  Chunks aren't optimal either.  I'd suggest losing both.